repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/apivalidator/multitenant/headerAllAtMethodTenantExplicit.wadl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<application xmlns="http://wadl.dev.java.net/2009/02"
+             xmlns:rax="http://docs.rackspace.com/api"
+             xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <resources base="https://test.api.openstack.com">
+        <resource path="/v1/resource" rax:roles="a:admin/{X-TENANT}">
+            <param name="X-OTHER"
+                   style="header"
+                   required="true"
+                   type="xsd:string"
+                   repeating="false"/>
+            <method name="POST" rax:roles="a:creator/{X-TENANT}">
+                <request>
+                    <param name="X-TENANT"
+                           style="header"
+                           required="true"
+                           type="xsd:int"
+                           repeating="true"
+                           rax:isTenant="true"/>
+                    <param name="X-TENANT"
+                           style="header"
+                           required="true"
+                           type="xsd:boolean"
+                           repeating="true"
+                           rax:isTenant="true"/>
```

Comment:
It may come with the documentation, but what's this boolean header param about?

Reply:
boolean header means that the validator will only allow the value of X-TENANT to be true or false.  The other param only allows ints.  So this is saying the values of X-TENANT should be either ints or boolean values -- will fail if that's not the case.

Reply:
Ah, I see. So it's just a way of validating the content. For the test that uses thing config, we only seem to be providing int tenant values, so is the boolean type parameter necessary? I would think not?

Reply:
I would think not.  I think this is derived from tests on my side and there I did a sanity check on bool vs int.

Reply:
Yes. It is based on this [API Checker test suite](https://github.com/rackerlabs/api-checker/blob/api-checker-2.6.0/core/src/test/scala/com/rackspace/com/papi/components/checker/ValidatorWADLRaxRolesHeaderMultiValueTenantSuite.scala).

Reply:
I see. So I'm always a fan of scoping tests to just what we care about because it makes them easier to grok and maintain. Plus it helps isolate the functionality under test. I kind of want to suggest we drop that second parameter for this test suite, even though it is copied from the `api-checker` suite, because we unlike the `api-checker` suite, we don't need it.

But I won't complain too much about that.